docs/_config.yml

```
@@ -0,0 +1,132 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole site, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing these this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.github_repo }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: gRPC-Gateway
+description: gRPC-Gateway Documentation Website
+# baseurl: "/just-the-docs" # the subpath of your site, e.g. /blog
+# url: "https://pmarsceill.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+
+repository: iamrajiv/grpc-gateway
+remote_theme: pmarsceill/just-the-docs
+
+permalink: pretty
+# exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile"
+# , "docs/tests/"
+# ]
```

Copy this from the other file
```suggestion
exclude: ["run.sh"]
```